pkg/api/unversioned/group_version.go

```diff
@@ -241,6 +253,28 @@ func (gv *GroupVersion) UnmarshalText(value []byte) error {
 	return gv.unmarshal(value)
 }
 
+// GroupVersions can be used to represent a set of desired group versions.
+type GroupVersions []GroupVersion
+
+// VersionForGroup implements the same interface as GroupVersions to allow easy
+// selection of group versions.
+func (gvs GroupVersions) VersionForGroup(group string) (GroupVersion, bool) {
```

Comment:
declare an actual interface here  for `VersionForGroup` and `PrefersGroup`.  Otherwise I think its less than obvious.